app/api/common/serializers.py

```
@@ -174,12 +174,28 @@ def build_ui_snapshot(app, base_url: str) -> dict[str, Any]:
+        "group": {"volume": 0, "muted": False},
+        "port": settings.sendspin_port,
+        "enabled": settings.sendspin_enabled,
+    }
+    if sendspin and sendspin.is_running:
+        try:
+            sendspin_data.update({
+                "clients": sendspin.list_clients(),
+                "group": sendspin.get_group_state(),
+            })
+        except Exception:
+            pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)